app/src/main/java/me/ccrama/redditslide/Activities/FullscreenImage.java

```diff
@@ -36,7 +36,7 @@
 /**
  * Created by ccrama on 3/5/2015.
  */
-public class FullscreenImage extends BaseActivity {
+public class FullscreenImage extends FullScreenActivity {

```

Comment:
So it won't have to be written in every full screen activity but in the full screen activities only